packages/core/src/components/search/Filter/FilterSlider.tsx

```diff
@@ -142,13 +139,11 @@ function FilterSlider({
     toggleFacet: onFacetChange,
     fallbackToFirst: true,
     allFacets: facets,
-    deliverySettings,
+    deliveryPromiseSettings,
   })
 
-  const regionalizationData = getRegionalizationSettings({
-    deliverySettings,
-  })
-  const { deliverySettings: deliverySettingsData } = regionalizationData
+  const cmsData = getGlobalSettings()
```

Comment:
Not necessary to send deliverySettings, because the only info that can come from a section is related to **regionalization** (`inputField` or `idkPostalCodeLink`) -- the idea of `getGlobalSettings` is to receive the global settings and, if there were already some section settings, `getGlobalSettings` merges them (global and section). The merge only happens for `RegionModal` and `RegionPopover` because they're the sections that still have those deprecated settings.